src/app/components/node/node.component.ts

```diff
@@ -39,9 +40,9 @@ export class NodeComponent implements OnInit, OnChanges, OnDestroy {
   gradientId = 'uninitialized';
 
   summaryColumns = [
+    { label: 'Type', cssSuffix: 'type', accessor: (severity: Severity, info: VariantInfo) => 'SNV' },
```

Comment:
@cholson are we sure we want Type first?

Reply:
yep. this was a request from charles

Reply:
thanks, and sorry, I see that now.

I'm double-checking with him that his preference remains the same even after we've added the severity column